website/docs/welcome.md

```diff
@@ -0,0 +1,33 @@
+---
+id: welcome
+title: Welcome
+---
+
+## Eightshift Development Kit
+
+Eightshift Development Kit contains all the tools you need to start building a modern WordPress theme, using all the latest front end and backend development tools.
+
+Lets brag a little about what you can expect from this setup:
+- **Custom Gutenberg blocks** - Easily use our pre-made 100% customizable existing Gutenberg blocks & components or create your own. We focus on **dynamic blocks** for developers.
+- **OOP** - All code is written following the latest object-oriented programming paradigms (abstract classes, interfaces, dependency injection, namespacing).
+- **WP-CLI** - Build your application with all-ready pre-created features that you can access with a single command using terminal.
+- **Webpack** - Modern assets & building process that you can customise.
+- **SASS** - Easy to learn CSS on steroids.
+- **Autoprefixing** - You don't have too worry is your code work on a older browsers.
```

Comment:
I think you have some typos here. Did you mean something like "You don't have to worry if your code will work on older browsers."?